lib/server.js

```diff
@@ -2,10 +2,14 @@ const express = require("express");
 const Number = require("./number");
 const pandoc = require("./pandoc");
 const http = require("http");
+const {execSync} = require("child_process");
 
 var app = express()
 .get("/:doc", function(req, res, next) {
-	if (req.path.endsWith("/"))
+	if (req.path.endsWith("/")) {
+		try {
+			req.query["-T"] = execSync("git branch --show-current", {cwd: __dirname}).toString();
+		} catch(e) {}
```

Comment:
What's this magic doing?

Reply:
Prefix the page title with the branch name: "tc/2023-06-07 - OData Extension for Data Aggregation Version 4.0".

Reply:
Will that add "main - ..." to the output in the main branch?